wdl/workflows/hiphase/hiphase.wdl

```diff
@@ -112,9 +112,12 @@ task run_hiphase {
 		RuntimeAttributes runtime_attributes
 	}
 
+	# to handle single samples with very high depth, we had to increase memory to 3*gpu
+	# to handle cohorts with very high depth, we had to increase memory to 4*gpu
 	Int threads = 16
-	Int mem_gb = threads * 2
+	Int mem_gb = threads * 4
 	Int disk_size = ceil(size(vcfs, "GB") + size(reference, "GB") + size(bams, "GB") * 2 + 20)
+	String haplotags_param = if length(haplotagged_bam_names) > 0 then "--haplotag-file ~{id}.~{refname}.hiphase.haplotags.tsv" else ""
```

Comment:
Just verifying I understand the logic:

This is only enabled if haplotagging is enabled, and haplotagging is only enabled for single-sample calls?